### DISCUSSION TITLE
Add support for Angular Universal

### DISCUSSION THREAD

**Discussion by elopez89** [vercel/community#247] [Help] [answered]
Is it possible to deploy an angular universal application onto vercel? I was able to successful deploy the application but it isn't being server-side rendered. You can tell by viewing the page source here: https://painter-quoting-cg3amwvqi-ppg-services-engineering.vercel.app/.  Is there a specific way i need to configure vercel in order to get this wired up correctly?

Project Settings:

![Screen Shot 2021-09-22 at 4 45 21 PM](https://user-images.githubusercontent.com/19472648/134419261-3691439f-94d5-4f2f-915b-c608ea08e717.png)

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/vercel/issues/6755#issuecomment-1002333052

**Comment by rodolfojnn** [comment]
Any news for this? It would be really cool to have a template for Angular Universal in Vercel.

**Reply by amyegan** [reply]
While Vercel does support Angular in general, I'm not aware of zero-config support for Angular Universal specifically. I did find a solution that someone shared on dev.to about [how to deploy Angular Universal to Vercel](https://dev.to/jdgamble555/how-to-deploy-angular-universal-to-vercel-31d0), which includes some Serverless Function setup steps. And the [Built Output API](https://vercel.com/blog/build-output-api) opens up a lot more framework possibilities.
